lib/coinbase/createContract.ts

```diff
@@ -32,7 +32,15 @@ export async function createContract({
   const smartAccount = await cdp.evm.createSmartAccount({ owner: evmAccount });
 
   // Use the protocol SDK to generate calldata
-  const { parameters } = await create1155({ contract, token, account });
+  const { parameters, tokenSetupActions } = await create1155({ contract, token, account });
+
+  const adminPermissionCall = encodeFunctionData({
+    abi: zoraCreator1155ImplABI,
+    functionName: "addPermission",
+    args: [BigInt(1), smartAccount.address, BigInt(PERMISSION_BIT_ADMIN)],
+  });
```

Comment:
please move this call into `create1155` so it is consistent with where other `tokenSetupActions` are already being built.
- actual: appending additional args AFTER tokenSetupActions are built.
- required: update create1155 so the original args are updated with the new setup action.